docs/pages/admin-guide.mdx

```diff
@@ -677,17 +630,15 @@ server IP with the URL of the proxy server. In the example below, we've
 replaced the auth server IP with the proxy web endpoint `teleport-proxy.example.com:3080`.
 
 ```bash
-$ sudo tctl nodes add
+sudo sudo tctl nodes add
 
-The invite token: n92bb958ce97f761da978d08c35c54a5c
-Run this on the new node to join the cluster:
-teleport start --roles=node --token=n92bb958ce97f761da978d08c35c54a5c --auth-server=teleport-proxy.example.com:3080
```

Comment:
I modified this here but left the examples above with the hardcoded example since they provide an overview of the concept using those (which my understanding of your intent).